README.md

```diff
@@ -559,6 +527,287 @@ You can build a pipeline to use [FHIR converter](https://github.com/microsoft/FH
+
+### Features
+- Support anonymization methods for DICOM metadata including redact, keep, encrypt, cryptoHash, dateShift, perturb, substitute, remove and refreshUID.
+- Configuration of the data elements that need to be anonymized.
+- Configuration of the anonymization methods for each data element.
+- Ability to run the tool on premise to anonymize a dataset locally.
+
+### Build the solution
+Use the .Net Core 3.1 SDK to build DICOM Anonymization Tool. If you don't have .Net Core 3.1 installed, instructions and download links are available [here](https://dotnet.microsoft.com/download/dotnet/3.1).
+
+### Prepare DICOM Data
+You can prepare your own DICOM files as input, or use sample DICOM files in folder $SOURCE\DICOM\samples of the project.
+
+## Anonymize DICOM data: using the command line tool
+
+Once you have built the command line tool, you will find executable file Microsoft.Health.Dicom.Anonymizer.CommandLineTool.exe in the $SOURCE\DICOM\src\Microsoft.Health.Dicom.Anonymizer.CommandLineTool\bin\Debug|Release\netcoreapp3.1 folder.
+
+You can use this executable file to anonymize DICOM file.
+
+```
+> .\Microsoft.Health.Dicom.Anonymizer.CommandLineTool.exe -i myInputFile -o myOutputFile
+```
+
+### Use Command Line Tool
+The command-line tool can be used to anonymize one DICOM file or a folder containing DICOM files. Here are the parameters that the tool accepts:
+
+
+| Option | Name | Optionality | Default | Description |
+| ----- | ----- | ----- |----- |----- |
+| -i | inputFile | Required (for file conversion) | | Input DICOM file. |
+| -o | outputFile | Required (for file conversion) | |  Output DICOM file. |
+| -c | configFile | Optional |configuration.json | Anonymizer configuration file path. It reads the default file from the current directory. |
+| -I | inputFolder | Required (for folder conversion) |  | Input folder. |
+| -O | outputFolder | Required (for folder conversion) |  | Output folder. |
+| --validateInput | validateInput | Optional | false | Validate input DICOM file against value multiplicity, value types and format in DICOM specification. |
+| --validateOutput | validateOutput | Optional | false | Validate output DICOM file against value multiplicity, value types and format in DICOM specification. |
+
+> **[NOTE]**
+> To anonymize one DICOM file, inputFile and outputFile are required. To anonymize a DICOM folder, inputFolder and outputFolder are required.
+
+Example usage to anonymize DICOM files in a folder:
+```
+>.\Microsoft.Health.Dicom.Anonymizer.CommandLineTool.exe convert -I myInputFolder -O myOutputFolder -c myConfigFile
+```
+
+## Sample configuration file
+The configuration is specified in JSON format and has three required high-level sections. The first section named _rules_, it specifies anonymization methods for DICOM tag. The second and third sections are _defaultSettings_ and _customizedSettings_ which specify default settings and customized settings for anonymization methods respectively.
```

Comment:
We need to unify all "_customizedSettings_" into "_customSettings_" both in text body and config samples.